Task2/config/constants.js

```diff
@@ -0,0 +1,17 @@
+const NOTFOUND = '404';
+const ErrorMessages = {
+  NOUSERS: 'There is no users to show',
+  NOTFOUND: 'Not found',
+  NOUSER: "User with such id isn't registered",
+  USERDELETED: 'User has been deleted successfully',
+  USERCREATED: 'User created successfully',
+  LOGINREQUIRED: 'Login field is required',
+  PASSWORDREQUIRED: 'Password field is required',
+  AGEREQUIRED: 'Age field is required',
+  AGEVALIDATE: 'Age should be between 4 and 130',
+};
+
+module.exports = {
```

Comment:
`module.exports` isn't  generally used when you're using ES6. Try using `export {errorStatus: NotFound, errorMessages: ErrorMessages}` over your current approach.

Reply:
Done